services/director-v2/src/simcore_service_director_v2/utils/osparc_variables.py

```diff
@@ -89,7 +89,7 @@ def copy(
         return {k: self._variables_getters[k] for k in selection}
 
 
-_HANDLERS_TIMEOUT: Final[NonNegativeInt] = parse_obj_as(NonNegativeInt, 4)
+_HANDLERS_TIMEOUT: Final[NonNegativeInt] = TypeAdapter(NonNegativeInt).validate_python(4)
```

Comment:
maybe this one owls as well.
```suggestion
_HANDLERS_TIMEOUT: Final[NonNegativeInt] = 4
```